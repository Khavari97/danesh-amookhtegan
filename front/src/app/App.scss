@import 'assets/fonts/fonts';

* {
  margin: 0;
  font-family: "Shabnam", serif;
}

a {
  text-decoration: none;
}

//Slider css
.alice-carousel__dots {
  .alice-carousel__dots-item:not(.__custom).__active {
    background-color: #046B00;
  }
}


