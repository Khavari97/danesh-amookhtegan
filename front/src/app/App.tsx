import React from "react";
import "./App.scss";
import AppRouter from "./navigation/AppRouter";
import {SnackbarProvider} from 'notistack';
import {SnackbarUtils} from "./components/material-ui/Snackbar/SnackbarUtils";
import ThemeProvider from "./assets/styles/material/material";

export interface AppProps {
}

export interface AppState {
}

class App extends React.Component<AppProps, AppState> {
    constructor(props: AppProps) {
        super(props);
        this.state = {};
    }

    componentDidMount = () => {
    }

    render() {
        return (
            <SnackbarProvider maxSnack={3} anchorOrigin={{horizontal: 'right', vertical: 'bottom'}}>
                <SnackbarUtils/>
                <ThemeProvider>
                    <AppRouter/>
                </ThemeProvider>
            </SnackbarProvider>
        );
    }
}

export default App;
