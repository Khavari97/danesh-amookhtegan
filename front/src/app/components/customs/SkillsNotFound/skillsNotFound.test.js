import React from "react"
import Enzyme, { shallow,configure,mount } from "enzyme";
import renderer from "react-test-renderer";
import App from './SkillsNotFound';
import Adapter from 'enzyme-adapter-react-16'
import {Typography} from '@material-ui/core';

configure({ adapter: new Adapter() })

const setup = (props = {}) => {
  return shallow(<App {...props} />);
};

const findByTestAttr = (wrapper, val) => {
    return wrapper.find(`[data-test="${val}"]`);
};

describe('render skillsNotFound component without error',()=>{
    let wrapper = setup();

    it('it should have value', () => {
          expect(wrapper.find(Typography).text()).toBe(' مهارتی یافت نشد!');
          expect(wrapper.find(Typography).prop('color')).toBe('primary');
          expect(wrapper).toBeTruthy();
          expect(wrapper.length).toEqual(1);
    });

    it('snapshot',()=>{
        const shot = <App />
        const snapshot = renderer.create(shot).toJSON();
        expect(snapshot).toMatchSnapshot();
    })
})