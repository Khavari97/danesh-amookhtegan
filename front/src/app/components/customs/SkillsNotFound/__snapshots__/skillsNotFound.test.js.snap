// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render skillsNotFound component without error snapshot 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  style={
    Object {
      "marginTop": "50px",
    }
  }
>
  <svg
    height="40"
    viewBox="0 0 65 65.235"
    width="40"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Group 5"
      id="Group_5"
      transform="translate(-1574 -446)"
    >
      <path
        d="M7.531,0A7.4,7.4,0,0,0,0,7.531V52.718a7.4,7.4,0,0,0,7.531,7.531H37.42a17.4,17.4,0,0,1-7.06-5.021H7.531a2.368,2.368,0,0,1-2.51-2.51V10.041H55.228V30.36a15.856,15.856,0,0,1,5.021,7.06V7.531A7.4,7.4,0,0,0,52.718,0Zm5.021,20.083V25.1h5.021V20.083Zm10.041,0V25.1H47.7V20.083ZM12.552,30.124v5.021h5.021V30.124Zm10.041,0v5.021h6.041a13.714,13.714,0,0,1,4.236-5.021Zm21.338,0a13.807,13.807,0,1,0,0,27.614A13.611,13.611,0,0,0,52.4,54.679l.471.471a2.356,2.356,0,0,0,.314,3.059l6.276,6.276a2.452,2.452,0,0,0,3.53,0L64.25,63.23a2.361,2.361,0,0,0,0-3.452L57.974,53.5a2.4,2.4,0,0,0-2.9-.471l-.471-.471a13.635,13.635,0,0,0,3.138-8.629A13.93,13.93,0,0,0,43.931,30.124Zm0,3.766A10.041,10.041,0,1,1,33.89,43.931,10.074,10.074,0,0,1,43.931,33.89Zm-31.38,6.276v5.021h5.021V40.166Zm10.041,0v5.021h3.766V43.931a19.014,19.014,0,0,1,.471-3.766Z"
        fill="#046b00"
        id="icons8_search_property"
        transform="translate(1574 446)"
      />
      <g
        data-name="Group 4"
        id="Group_4"
        transform="translate(162 113)"
      >
        <line
          data-name="Line 3"
          fill="none"
          id="Line_3"
          stroke="#046b00"
          strokeLinecap="round"
          strokeWidth="2"
          transform="translate(1450.5 372.5)"
          x2="10"
          y2="9"
        />
        <line
          data-name="Line 4"
          fill="none"
          id="Line_4"
          stroke="#046b00"
          strokeLinecap="round"
          strokeWidth="2"
          transform="translate(1450.5 372.5)"
          x1="10"
          y2="9"
        />
      </g>
    </g>
  </svg>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-colorPrimary"
    style={
      Object {
        "marginRight": "10px",
      }
    }
  >
     مهارتی یافت نشد!
  </p>
</div>
`;
